src/oracle/query_dsl/returning.rs

```diff
@@ -0,0 +1,104 @@
+use diesel::associations::HasTable;
+use diesel::query_builder::InsertStatement;
+use diesel::query_builder::{AstPass, Query, QueryFragment};
+use diesel::query_source::{Column, Table};
+use diesel::query_dsl::RunQueryDsl;
+use diesel::result::QueryResult;
+
+use oracle::backend::Oracle;
+use oracle::connection::OciConnection;
+
+pub trait OciReturningDsl {
+    type Output;
+
+    fn oci_returning(self) -> Self::Output;
```

Comment:
We need to call this function in our internal code.
So the implementation of `GstInsertable` should look similar to:

```rust
::diesel::insert_into(table).values(insertable).oci_returning().get_result(conn)
```